config/idms/file/FixedLinks_v1.0.ts

```diff
@@ -0,0 +1,72 @@
+import {XmlFile} from "../../../src/feed/file/XmlFile";
+import {ParsedRecord, RecordAction} from "../../../src/feed/record/Record";
+import {ALFRecord} from "../../timetable/file/ALF";
+
+class FixedLinkRecord extends ALFRecord {
+    public readonly name: string = "idms_fixed_link";
+
+    public extractValues(link: any): ParsedRecord {
+        if (typeof link.daysofweek === 'string') {
+            link.daysofweek = [link.daysofweek];
+        }
+        if (!link.daysofweek.filter) {
+            console.log(link);
+            process.exit(0);
```

Comment:
Maybe I misunderstood something, but config file should exit process I guess... ...